app/client/app.jsx

```diff
@@ -22,7 +22,7 @@ const composeSetup = process.env.NODE_ENV !== 'prod' && typeof window === 'objec
   window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ ?
   window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ : compose
 
-const store = createStore(
+export const store = createStore(
```

Comment:
@tristanhcole this seems to be exported but not used anywhere?

Reply:
oh yeah I ended up storing the client org ID in the browser local storage ...

Reply:
so doesn't need to be exported anymore...

Reply:
ok will remove 👍